<launch>
  <rosparam ns="models">
    ['Amicelli','Rusk_Box','Soda_Club_Bottle']
  </rosparam>
  <group ns="model_paths">
    <param name="Amicelli" value="package://objrec_ransac/ObjRecRANSAC/data/Soda_Club_Bottle.vtk"/>
    <param name="Rusk_Box" value="package://objrec_ransac/ObjRecRANSAC/data/Rusk_Box.vtk"/>
    <param name="Soda_Club_Bottle" value="package://objrec_ransac/ObjRecRANSAC/data/Soda_Club_Bottle.vtk"/>
  </group>
</launch>
